README.md

```diff
@@ -49,7 +49,7 @@ Among the FREE features of each admin interface:
 - [Reorder](https://backpackforlaravel.com/docs/crud-operation-reorder) operation
 - [Revise](https://backpackforlaravel.com/docs/crud-operation-revisions) operation (aka. audit log)
 
-But professionals don't love Backpack just because it's feature-packed. They also love it because it's **ridiculously easy to override a functionality**. Generally, you just need to create a function (or blade file) with the right name. Yes, it can be _that_ easy.
+But professionals love the Backpack not only because it is multifunctional. They also love it because it's **ridiculously easy to override a functionality**. Generally, you just need to create a function (or blade file) with the right name. Yes, it can be _that_ easy.
```

Comment:
```suggestion
But professionals love it not only because it is feature-packed. They also love it because it's **ridiculously easy to override a functionality**. Generally, you just need to create a function (or blade file) with the right name. Yes, it can be _that_ easy.
```